template.yml

```diff
@@ -363,25 +375,41 @@ Resources:
                 httpMethod: POST
                 type: aws_proxy
         components:
-          schemas:
+          AccessRight:
+            type: string
+            enum:
+              - approve_doi_request
+              - reject_doi_request
             Role:
               properties:
                 type:
                   type: string
                 rolename:
                   type: string
+                accessRight:
+                  type: array
+                  items:
+                    $ref: '#/components/schemas/AccessRight'
             User:
               properties:
                 type:
                   type: string
                 username:
                   type: string
+                givenName:
+                  type: string
+                familyName:
+                  type: string
                 institution:
                   type: string
                 roles:
                   type: array
                   items:
                     $ref: '#/components/schemas/Role'
+                accessRight:
```

Comment:
AccessRight on both Role and User?

For reading purposes only I assume?

Reply:
For user only for reading purposes.